tensorflow_addons/optimizers/weight_decay_optimizers.py

```diff
@@ -71,7 +72,12 @@ def __init__(self, weight_decay, *args, **kwargs):
     """
 
     @typechecked
-    def __init__(self, weight_decay: Union[FloatTensorLike, Callable], **kwargs):
+    def __init__(
+        self,
+        weight_decay: Union[FloatTensorLike, Callable],
+        exclude_from_weight_decay: Optional[List[str]] = None,
```

Comment:
Please add this to the Args section below.

Reply:
Added.

Reply:
Thanks! Could you also add a sentence that explains that decay_var_list in minimize takes priority over exclude_from_weight_decay if specified (and also add a corresponding sentence to the documentation to minimize)?

Reply:
Updated `DecoupledWeightDecayExtension` `__init__`, `minimize` and `apply_gradients` docs, also `extend_with_decoupled_weight_decay` doc. Added `exclude_from_weight_decay` in `AdamW` and `SGDW` `**kwargs` doc.